### ISSUE TITLE
would be nice if graph.multiply supported broadcasting more generally

### ISSUE THREAD

**Issue by silky** [PAIR-code/deeplearnjs#113]
i get the following error trying to do a multiply on two tensors:

```
graph.multiply(a, b)

...

Uncaught Error: Error adding multiply op: one of inputs must be
 scalar or the shapes 1,1,32 and 65536,1 must match.
```

in tensorflow, this is actually supported, and the resulting shape is 1,65536,32

**Comment by nsthorat** [comment]
Ahh yes, I'm sorry. BTW, I checked out your CPPN demo, if you're not training you should not use the Graph layer and just use the NDArrayMath layer. This layer support broadcasting in the way you want.

**Comment by silky** [comment]
thanks; but yes, i plan on adding some training, hence using the graph one :)

no worries, i realise this library is new!

**Comment by dsmilkov** [comment]
Hi,

We [released 0.5.0](https://github.com/PAIR-code/deeplearnjs/releases/tag/v0.5.0) and we are deprecating the graph API in favor of the new eager API which does support broadcasting.

Closing this issue accordingly. Feel free to open another issue regarding the eager API if you have any problems with broadcasting.

Thanks!
